# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/thunderbird/hotfix-icuuc.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- thunderbird-135.0.1/js/moz.configure.vanilla	2025-03-06 11:54:12.944100451 +0100
+++ thunderbird-135.0.1/js/moz.configure	2025-03-06 11:54:48.558264923 +0100
@@ -1301,7 +1301,7 @@
     return enable_system_icu
 
 
-system_icu = pkg_check_modules("MOZ_ICU", "icu-i18n >= 73.1", when="--with-system-icu")
+system_icu = pkg_check_modules("MOZ_ICU", "icu-i18n >= 73.1, icu-uc >= 73.1", when="--with-system-icu")
 
 
 @depends(enable_system_icu_option)
